### PR TITLE
Docs: Fix minor issues and add typings to every TS snippet

### PR DESCRIPTION
**Pull request by Mad-Kat**
Minor fixes:
- Remove empty space at the beginning of tags
- Corrected output 

Added `twoslash` to every TypeScript snippet

Added atoms & Tailwind to features

### CODE REVIEW

File: packages/docs/docs/pages/features.md [hunk 1]
```diff
@@ -133,32 +134,87 @@ const Paragraph = styled.p`
...
+    style: { '--var1': props => props.$primary ? "#BF4F74" : "white" }
   }
 );
 ```
 
 ```css [output CSS]
 .yakClass1 {
   background: var(--var1);
+
+  &:where(.yakCSSClass1) {
```
Comment (jantimon) [inline packages/docs/docs/pages/features.md:150]: &:where(.primary) {

File: packages/docs/docs/pages/why-next-yak.md [hunk 2]
```diff
@@ -127,8 +128,9 @@ export const MyComponent = (props) => {
 ```
 
 
-```tsx [typescript]
-import { FC } from 'react';
+```tsx twoslash [typescript]
+// @noErrors
+import { FC, ReactNode } from 'react';
```
Comment (jantimon) [inline packages/docs/docs/pages/why-next-yak.md:133]: is FC still the way to go?

File: packages/docs/docs/pages/features.md [hunk 3]
```diff
@@ -100,8 +101,8 @@ const Paragraph = styled.p<{ $primary?: boolean }>`
 const Component = () => {
   return (
     <>
-      <Paragraph $primary> Hello there primary!</Paragraph>
-      <Paragraph> Hello there non-primary!</Paragraph>
+      <Paragraph $primary>Hello there primary!</Paragraph>
+      <Paragraph >Hello there non-primary!</Paragraph>
```
[the remark this answers is not in the document]
Comment (Mad-Kat) [reply packages/docs/docs/pages/features.md:105]: Nope 😄 

Thanks for the hint

File: packages/docs/docs/pages/why-next-yak.md (hunk 2 above)
Comment (Mad-Kat) [reply packages/docs/docs/pages/why-next-yak.md:133]: I think so. Or do you mean `FunctionComponent`? Or anything else?

**Review by jantimon — APPROVED** [review]

**Pull request merged**